src/javascript/app/pages/trade/charts/highchart.js

```diff
@@ -13,6 +13,7 @@ const addComma         = require('../../../common/currency').addComma;
 const localize         = require('../../../../_common/localize').localize;
 const State            = require('../../../../_common/storage').State;
 const getPropertyValue = require('../../../../_common/utility').getPropertyValue;
+const isEmptyObject = require('../../../../_common/utility').isEmptyObject;
```

Comment:
spacing

Reply:
noted and addressed, please check [#796ab91](https://github.com/binary-com/binary-static/commit/796ab91735b5c749bd80fda5fda969a80bd8db04)